libs/components/src/radio/stories/vl-radio.stories-arg.ts

```diff
@@ -17,43 +17,43 @@ export const radioArgTypes = {
         description: 'Attribute to show the checkbox as a block element',
```

Comment:
Kan je dit vertalen naar het Nederlands en de "Attribute to..." wegwerken?

Reply:
done